server/endpoint/creator/error.go

```diff
@@ -1,4 +1,4 @@
-package healthz
+package creator
```

Comment:
Diff is weird. Not certain why. Maybe I didn't commit the healthz removal correctly earlier?